.github/workflows/unit_tests.sh

```diff
@@ -64,6 +68,33 @@ for phase in "${PHASES[@]}"; do
             # Never remove halt_on_error from UBSAN_OPTIONS. See https://github.com/systemd/systemd/commit/2614d83aa06592aedb.
             export UBSAN_OPTIONS=print_stacktrace=1:print_summary=1:halt_on_error=1
 
+            # FIXME
+            # For some strange reason the GH Actions VM stops responding after
+            # executing first ~150 tests, _unless_ there's something producing
+            # output (either running `meson test` in verbose mode, or something
+            # else in background). Despite my efforts so far I haven't been able
+            # to identify the culprit (since the issue is not reproducible
+            # during debugging, wonderful), so let's at least keep a workaround
+            # here to make the builds stable for the time being.
+            (set +x; while :; do echo -ne "\n[WATCHDOG] $(date)\n"; sleep 30; done) &
+            meson test --timeout-multiplier=3 -C build --print-errorlogs
+            ;;
+        RUN_CLANG_ASAN_RUST)
+            MESON_ARGS=(--optimization=1)
+
+            export CC=clang
+            export CXX=clang++
+            # Build fuzzer regression tests only with clang (for now),
+            # see: https://github.com/systemd/systemd/pull/15886#issuecomment-632689604
+            # -Db_lundef=false: See https://github.com/mesonbuild/meson/issues/764
+            MESON_ARGS+=(-Db_lundef=false -Dfuzz-tests=true)
+            MESON_ARGS=(-Dbuild-rust=true)
```

Comment:
```suggestion
            MESON_ARGS+=(-Dbuild-rust=true)
```

This should, hopefully, fix the clang fail.

Reply:
d'oh! good catch